src/signal.rs

```diff
@@ -1706,34 +1707,111 @@ mod editable {
     }
 }
 
+pub trait SpinButtonSignals {
```

Comment:
If it's in this file, then yes. Want me to do it in this PR as well? Or I open a new one?